stacks/dovetail.prx.org.yml

```diff
@@ -509,6 +512,46 @@ Resources:
           Value: !Ref AWS::StackName
         - Key: "prx:cloudformation:stack-id"
           Value: !Ref AWS::StackId
+  # connect dovetail-router impressions to ddb kinesis
+  DovetailRouterSubscriptionRole:
```

Comment:
Can you add a `Condition: CreateStagingResources` to this for now?  Just in case!